rest/views.py

```diff
@@ -0,0 +1,134 @@
+"""
+Controllers for REST app
+"""
+
+from __future__ import unicode_literals
+
+from rest_framework.generics import (
+    ListCreateAPIView,
+    RetrieveUpdateDestroyAPIView,
+)
+from rest_framework.permissions import (
+    DjangoObjectPermissions,
+    IsAuthenticated,
+)
+from rest_framework.reverse import reverse
+
+from rest.serializers import (
+    VocabularySerializer,
+    RepositorySerializer,
+    TermSerializer,
+)
+from learningresources.models import Repository
```

Comment:
`rest` is the new app being created in this PR so this should properly separate our imports from the django/rest_framework imports